spec/services/back_end_service_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe BackEndService do
-  it 'can retrieve the base url for API calls' do
+  xit 'can retrieve the base url for API calls' do
     expect(BackEndService.base_url).to eq 'https://spotme-app-api.herokuapp.com'
   end
```

Comment:
Personally, I'd prefer to merge into `main` with the Heroku `base url` so the deployed Heroku site is functioning.  I think we should update this before merging to `main`.  But open to hearing what other teammates think.